rfc/graphql-schema/additionalTypeDefs/byPage/shoppingListDetails.ts

```diff
@@ -0,0 +1,143 @@
+export default /* GraphQL */ `
+  type ShoppingListItem implements Node {
+    id: ID!
+    configuredProductId: ID!
+    name: String!
+    sku: String!
+    price: Money!
+    imageUrl: String!
+    quantity: Int!
+    productOptions: [OrderLineItemProductOption!]!
```

Comment:
don't we need the full `CartSelectedOption`?

since we need to render option id / value id for edit view?

Reply:
added